coreos-4/policies/MP-Media_Protection/component.yaml

```diff
@@ -215,19 +220,11 @@
 - control_key: MP-6 (8)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: not applicable
```

Comment:
This seems applicable, but CoreOS and OpenShift would not meet the requirement due to not having remote wipe/purge capabilities.

Reply:
How's it applicable if you can't do it? And we went through assessment and determined that it is not applicable?

Reply:
Would be ``unsatisfied`` and deferred to third party ISV.

Reply:
Actually.... I am wrong. Reread the control just now:

`````

The organization provides the capability to purge/wipe information from [Assignment: organization-defined information systems, system components, or devices] either remotely or under the following conditions: [Assignment: organization-defined conditions].
`````

Requirement is for the *organization* to provide this capability, not the information system or information system component. This should be not applicable.